### PR TITLE
Feature - Add Additional Intro Labs

### PR DESCRIPTION
**Pull request by jkeam**
Add additional intro labs, in particular how it works and installation.  Also corrected some previous intro content.

### CODE REVIEW

**Review by theckang — COMMENTED** [review]
Can we update the `serverless_create_subscription.png` image with `Manual` selected?

File: workshop/lab1.3_configure.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+```
+
+Wait a while and the output should look like the following
+```
+DependenciesInstalled=True
+DeploymentsAvailable=True
+InstallSucceeded=True
+Ready=True
+```
+
```
Comment (theckang) [inline workshop/lab1.3_configure.md]: Can we add a "verify pods" step similar to knative eventing?

File: workshop/lab1.1_welcome.md [hunk 2]
```diff
@@ -22,4 +22,4 @@ Red Hat OpenShift Serverless is built upon the Red Hat OpenShift platform, which
...
 
-Red Hat OpenShift Serverless not only opens serverless architecture, but also adds monitoring, metering, A/B testing, canary releases, and publicly exposed routes.  Not only that, but another benefit of it running within the OpenShift ecosystem is that all of the hybrid cloud and cloud native development tools are available and all integrate and work well with each other.  This includes things like CodeReady Workspaces, supported runtimes for of your container, and thousands of integration connectors ready to go out of the box like Kafka event streams.
+Red Hat OpenShift Serverless not only opens serverless architecture, but also adds monitoring, metering, A/B testing, canary releases, and publicly exposed routes.  Not only that, but another benefit of it running within the OpenShift ecosystem is that all of the hybrid cloud and cloud native development tools are available and all integrate and work well with each other.  This includes things like CodeReady Workspaces, supported runtimes for of your container, and hundreds of integration connectors ready to go out of the box like Kafka event streams.
```
Comment (dudash) [inline workshop/lab1.1_welcome.md]: "runtimes for of your container"

Comment (dudash) [reply workshop/lab1.1_welcome.md]: Suggest changing the second sentence to remove the "Not only that" with a "And another benefit..."

File: workshop/lab1.2_how_works.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# How It Works
+
+Red Hat OpenShift Serverless delivers Kubernetes-native, event-driven primitives for microservices, containers, and compatible Function-as-a-Service (FaaS) implementations. OpenShift Serverless provides out-of-the-box traffic routing and security capabilities. This offering combines Red Hat Operators, Knative, and Red Hat OpenShift. Combined, these tools allow stateless and serverless workloads to run across OpenShift deployments on private, public, hybrid, or multi-cloud environments with automated operations. But how exactly does this work?
+
+## Architecture
+At a high level, Serverless -- specifically the Knative project -- is composed of two main components: eventing and Serving.  These two components work together to create an event driven architecture that is a critical part of the Serverless paradigm.
```
Comment (dudash) [inline workshop/lab1.2_how_works.md]: capitalize Eventing or lowercase Serving

File: workshop/lab1.2_how_works.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+![OpenShift Serverless Architecture](content/images/serverless_eventing_serving.png)*Architecture*
+
+### Eventing
+Eventing is the part of the infrastructure for consuming external events and producing a cloud specific event called a CloudEvent.  A CloudEvent is a CNCF specfication for describing cloud event data in a common way.  It seaks to be consistent, accessible across a variety of languages, and portable.  This eventing abstraction is a useful one as it allows developers to only have to think about functional areas of the system that concerns them.  For instance, thinking about applications in an evented way allows for loosely coupled, independent services.  This also allows developers to work on specific aspects of a given application in a variety of ways:
```
Comment (dudash) [inline workshop/lab1.2_how_works.md]: replace seaks with seeks

File: workshop/lab1.3_configure.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+Next make sure to install in all namespaces.  Select the latest Update Channel made available to you.  For the purposes of this demo, we'll pick the Manual approval strategy.  Lastly, click Subscribe.
+
+![Subscribe to OpenShift Serverless Operator](content/images/serverless_create_subscription.png)*Subscribe*
+
+Wait a few minutes and eventually you will see it show up as Installed Operator with the status of Succeeded
+
+![Subscribe to OpenShift Serverless Operator](content/images/serverless_installed_operator.png)*Installed*
+
+
+## Knative Serving Installation
```
Comment (dudash) [inline workshop/lab1.3_configure.md]: any reason these are CLI and the operator is via webconsole? Seems a little odd - if we are doing it to showcase the flexibility to use webconsole or CLI, I think we should call that out with a quick little blurb:
"OpenShift Serverless install can be completely done with Operator Hub webconsole or the CLI - we are going to give you a little taste of both by adding the operator via webconsole and then the rest via CLI"

**Review by dudash — CHANGES_REQUESTED** [review]
I added some comments inline  - Dudash

File: workshop/lab1.3_configure.md (hunk 5 above)
Comment (theckang) [reply workshop/lab1.3_configure.md]: In the future, we will need to pin our workshop to a specific version of Serverless.  So, we'll have to use the `ClusterServiceVersion` API.   I actually think we should do this now.

File: workshop/lab1.3_configure.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+imc-controller-5b75d458fc-ptvm2        1/1     Running   0          7m19s
+imc-dispatcher-64f6d5fccb-kkc4c        1/1     Running   0          7m18s
+```
+
+
+## `kn` CLI Installation
+
+`kn` is a very powerful tool for being able to control knative from the command line.
+
+1.  Download the [CLI](https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest)
```
Comment (dudash) [inline workshop/lab1.3_configure.md]: They could also get it from the CLI Tools page in their cluster:
e.g. for CRC it's at https://console-openshift-console.apps-crc.testing/command-line-tools

Comment (dudash) [reply workshop/lab1.3_configure.md]: not sure it really matters, but maybe a little cooler

**Comment by theckang** [comment]
This update is outside the `content` folder so we have duplicate files, and it won't render correctly in homeroom.  Can we merge the new files into `content` folder?

File: workshop/lab1.3_configure.md (hunk 1 above)
Comment (jkeam) [reply workshop/lab1.3_configure.md]: done

File: workshop/lab1.1_welcome.md (hunk 2 above)
Comment (jkeam) [reply workshop/lab1.1_welcome.md]: done

File: workshop/lab1.2_how_works.md (hunk 3 above)
Comment (jkeam) [reply workshop/lab1.2_how_works.md]: done

File: workshop/lab1.2_how_works.md (hunk 4 above)
Comment (jkeam) [reply workshop/lab1.2_how_works.md]: done

File: workshop/lab1.3_configure.md (hunk 6 above)
Comment (jkeam) [reply workshop/lab1.3_configure.md]: done

File: workshop/lab1.3_configure.md (hunk 5 above)
Comment (jkeam) [reply workshop/lab1.3_configure.md]: Requesting to do this in another PR.  I opened an issue to track it https://github.com/RedHatGov/serverless-workshop-dashboard/issues/3.

**Comment by jkeam** [comment]
@theckang Yep moved everything into the `content` folder.

**Review by dudash — APPROVED** [review]

**Pull request merged**